Makefile.common

```diff
@@ -202,6 +202,9 @@ common-unused:
 common-build: promu
 	@echo ">> building binaries"
 	$(PROMU) build --prefix $(PREFIX) $(PROMU_BINARIES)
+	@if [ "$(GOHOSTOS)" = "darwin" ] && command -v codesign > /dev/null 2>&1; then \
+		codesign --sign - --force --preserve-metadata=entitlements,requirements,flags,runtime $(PREFIX) >/dev/null 2>&1; \
```

Comment:
Could you just add the following `echo`, so the user can see that code signing happens? When executing it, I was wondering whether the signing was actually executing :)
```suggestion
		echo "Signing binaries..."
		codesign --sign - --force --preserve-metadata=entitlements,requirements,flags,runtime $(PREFIX) >/dev/null 2>&1; \
```